client-next/components/dashboard/link_tree_item.tsx

```diff
@@ -2,12 +2,14 @@ import { Link } from "../../near/types";
 import { useNear } from "../../context/near";
 
 interface Props {
+  onEditClick: (link: Link) => void;
   link: Link;
 }
 
-const LinkTreeItem = ({ link }: Props) => {
+const LinkTreeItem = ({ link, onEditClick }: Props) => {
```

Comment:
You could omit the world `click`.
```suggestion
const LinkTreeItem = ({ link, onEdit }: Props) => {
```